lib/fluent/supervisor.rb

```diff
@@ -385,6 +385,9 @@ def self.default_options
         standalone_worker: false,
         signame: nil,
         winsvcreg: nil,
+        winsvc_name: 'fluentdwinsvc',
```

Comment:
default_options is bad place for these options because
default_options is for Supervisor, not command line.
This is why test failed.

BTW, `winsvcreg` can be removed from `default_options`.
There is no use for now.